src/index.jsx

```diff
@@ -1,8 +1,10 @@
 import React from "react";
 import ReactDOM from "react-dom";
-
 import App from "./App";
 
+import "bootstrap/dist/css/bootstrap.min.css";
```

Comment:
all that border etc classes are coming from here !

Reply:
ok